src/Compilers/CSharp/Portable/Symbols/TypeSymbolWithAnnotations.cs

```diff
@@ -477,6 +467,31 @@ internal NullableAnnotation GetValueNullableAnnotation()
             }
         }
 
+        internal bool CanBeAssignedNull
```

Comment:
>CanBeAssignedNull [](start = 22, length = 17)

nit: why a property instead of a method (see `ValueCanBeNull()` above)?

Reply:
Eliminating ValueCanBeNull to remove the inconsistency.

---
In reply to: [260039580](https://github.com/dotnet/roslyn/pull/33648#discussion_r260039580) [](ancestors = 260039580)